docs/guide/features.md

```diff
@@ -30,7 +30,29 @@ ESM first, top level await
 
 ## Threads
 
-Workers multi-threading via [tinypool](https://github.com/Aslemammad/tinypool) (a lightweight fork of [Piscina](https://github.com/piscinajs/piscina))
+Workers multi-threading via [tinypool](https://github.com/Aslemammad/tinypool) (a lightweight fork of [Piscina](https://github.com/piscinajs/piscina)). Multi-threading helps us run tests simultaneously so Vitest does not let heavy tests block us from running other tests.
+
+```bash
+$ vitest --threads
+```
+
+Multi-threading is enabled by default, but it is also possible to disable this feature, where sometimes it can result in better performance.
+
+ ```bash
+$ vitest --no-threads
+```
```

Comment:
I would not recommend disabling threads at all. Because this disables isolation too. And threads without isolation it still faster then threads disabled.

Reply:
Agreed. Maybe we can tell folks that this is possible (removing the result in better performance part), How is that?

Reply:
Yeah, option should be documented, but we should not encourage using it. And we should point out that this disables isolation too.

Reply:
Pushed!